### DISCUSSION TITLE
The Apache Community | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#10] [Comments]
# The Apache Community | 夜天之书

大约二十年前我刚开始进入互联网的世界的时候，支撑起整个网络的基础设施，就包括了 Apache 软件基金会（ASF）治下的软件。 Apache Httpd 是开启这个故事的软件，巅峰时期有超过七成的市场占有率，即使是在今天 NGINX 等新技术蓬勃发展的时代，也有三成左右的市场占有率。由 Linux、Apache Httpd、MySQL 和 PHP 组成的 LAMP 技术栈，是开源吞噬软件应用的第一

https://tisonkun.org/2022/03/13/the-apache-community/

**Comment by wey-gu** [comment]
恭喜 Tison 👍🏻，为你开心！

在吴晟发起这个 issue 之前，Nebula Graph 社区其实对 Apache 2.0 + CC foo 意味着不是 Open Source 这件事情是没有认识的，我也非常庆幸最后社区决定改为 Apache 2.0 而不是其他 license。

**Reply by tisonkun** [reply]
谢谢！确实 ASF (Board) Member 的职责之一就是传播 Apache License 的理念和规范用法。最近我在关注 [bytebase 项目公司 Apache License + Commercial License 的实践](https://github.com/bytebase/bytebase/pull/558)，希望越来越成熟。
